include/dlaf/util_matrix.h

```diff
@@ -136,16 +136,27 @@ class getter_random<std::complex<T>> : private getter_random<T> {
 
 }
 
-/// Sets all the elements of all the tiles to zero
+/// Sets the subset of local tiles of @p matrix in the 2D range @p begin @p sz to zero.
```

Comment:
```suggestion
/// Sets to zero the subset of local tiles of @p matrix in the 2D range starting at @p begin with size @p sz.
```

Moreover (absolutely not for this PR), I think that it might be preferable to pass an `iterate_range2d` instead of boundaries/(start+size).

In that way we have "for free" all the overloads we have for building an `iterate_range2d`, and it will be also more straightforward the doc (e.g. just saying "in the range @p range"). Probably, it will be anyway better to wait until we will improve iterator to be described by global indices and return just local ones.

This is something that might be discussed during future API refactoring.

Reply:
https://github.com/eth-cscs/DLA-Future/pull/732/commits/b76a16a814a050995bcfc86a7f7c5cbba100f195